scripts/python/tao_build.py

```diff
@@ -155,6 +157,11 @@ def configure_compiler(root, args):
                 f.write("\n")
                 f.write("build --remote_cache={}\n".format(token))
                 f.write("test --remote_cache={}\n".format(token))
+        with open(".tf_configure.bazelrc", "a") as f:
+            f.write("\n")
+            bazel_startup_opts = "--host_jvm_args=-Djdk.http.auth.tunneling.disabledSchemes="
```

Comment:
Will this work without proxy environment variables?

Reply:
Yes.